pybossa/view/fileproxy.py

```diff
@@ -126,13 +126,13 @@ def read_encrypted_file(project_id, bucket, key_name, signature):
     return response
 
 
-@blueprint.route('/encrypted/<string:store>/<string:bucket>/<string:workflow>/<string:workflow_uid>/<path:path>')
+@blueprint.route('/encrypted/<string:store>/<string:bucket>/workflow_request/<string:workflow_uid>/<path:path>')
 @no_cache
 @login_required
-def encrypted_workflow_file(store, bucket, workflow, workflow_uid, path):
+def encrypted_workflow_file(store, bucket, workflow_uid, path):
     """Proxy encrypted task file in a cloud storage"""
     project_id = int(path.split('/')[0])
```

Comment:
1. can `project_id` be also parameter to the method so that this line is not required similar to `encrypted_file` method
2. docstring needs an update re. workflow